terraform/testcases/prometheus_sd/otconfig.tpl

```diff
@@ -0,0 +1,28 @@
+receivers:
```

Comment:
will we be able to use this config in ec2?

Reply:
No, this config is specific to kubernetes (as its doing service discovery on the Kubernetes objects).

Reply:
Do we need to get it working on EC2 in order to merge this PR? I have made the changes to get it working with EC2 and was going to submit the change in a PR for soaking/performance tests. I can move those changes to this PR if preffered

Reply:
no, we can tackle that in the next pr